change/@ni-nimble-components-f4776f02-4ee0-4534-8a00-bbc51b09d9c3.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
As you pointed out offline, the Safari test is rendering incorrectly in Chromatic but correctly in local tests with both newer and older versions of Safari. It'd be good to update the PR description to mention that we noticed this and decided to accept it (or do something else, if you come up with an alternate resolution).

Reply:
The fonts in the plain text spans around the buttons are also different across browsers in chromatic. Wonder if using bodyFont on them will make them more consistent in all the screenshots.

Can wrap the whole thing in a span and set the font on the wrapper span since font is inherited.